example/lib/main.dart

```diff
@@ -30,7 +30,7 @@ class MyHomePage extends StatefulWidget {
   final String title;
 
   @override
-  _MyHomePageState createState() => _MyHomePageState();
+  State<MyHomePage> createState() => _MyHomePageState();
```

Comment:
This was changed because of a new lint rule which recommends avoiding exposing private members through public APIs.